test/unit/gateways/realex_test.rb

```diff
@@ -51,6 +51,41 @@ def setup
     @amount = 100
   end
 
+  def test_initialize_sets_refund_and_credit_hashes
+    refund_secret = 'refund'
+    rebate_secret = 'rebate'
+
+    options = {
+      login: @login,
+      password: @password,
+      rebate_secret: rebate_secret,
+      refund_secret: refund_secret
+    }
+
+    gateway = RealexGateway.new(options)
```

Comment:
The other gateways in this file and in other places that I've seen pass in the variables directly to the `Gateway.new` call, which I think looks cleaner and is less confusing. It's also nice to be consistent.

```suggestion
    gateway = RealexGateway.new(
      login: @login,
      password: @password,
      rebate_secret: rebate_secret,
      refund_secret: refund_secret
    )
```

Reply:
Done.

Reply:
Thanks for making this change!